cmd/eth1indexer/main.go

```diff
@@ -280,8 +281,16 @@ func main() {
 
 			err = IndexFromNode(bt, client, int64(lastBlockFromBlocksTable)-*offsetBlocks, int64(lastBlockFromNode), *concurrencyBlocks)
 			if err != nil {
-				logrus.WithError(err).Errorf("error indexing from node, start: %v end: %v concurrency: %v", int64(lastBlockFromBlocksTable)-*offsetBlocks, int64(lastBlockFromNode), *concurrencyBlocks)
+				errMsg := fmt.Sprintf("error indexing from node, start: %v end: %v concurrency: %v", int64(lastBlockFromBlocksTable)-*offsetBlocks, int64(lastBlockFromNode), *concurrencyBlocks)
+				errEntry := logrus.WithError(err)
+				if time.Since(lastSuccessulBlockIndexingTs) > time.Minute*30 {
+					errEntry.Fatal(errMsg)
```

Comment:
@Eisei24 we should use your utils.LogError here (as I already merged)

Reply:
Merged master into the branch and changed this part here.